pyomo/core/tests/unit/test_suffix.py

```diff
@@ -1567,7 +1567,7 @@ def test_clone_ObjectiveArray(self):
         self.assertEqual(inst.junk.get(model.obj[1]), None)
         self.assertEqual(inst.junk.get(inst.obj[1]), 1.0)
 
-    def test_clone_ObjectiveData(self):
+    def test_cloneObjectiveData(self):
```

Comment:
This (and the other three changes in this file) looks like just a find-and-replace gone slightly awry.